pets/etcd/etcd-petset.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apps/v1alpha1
+kind: PetSet
+metadata:
+  name: etcd
+spec:
+  serviceName: "etcd"
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: etcd
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+    spec:
+      containers:
+      - name: etcd
+        image: gcr.io/google_containers/etcd-amd64:2.2.5
+        ports:
+        - containerPort: 2380
+          name: peer
+        - containerPort: 2379
+          name: client
+        env:
+        - name: CLUSTER_SIZE
+          value: "3"
+        - name: PETSET_NAME
+          value: "etcd"
+        volumeMounts:
+        - name: datadir
+          mountPath: /var/run/etcd
+        command:
+          - "/bin/sh"
+          - "-c"
+          - |
+            for i in $(seq 0 $((${CLUSTER_SIZE} - 1))); do
```

Comment:
Currently, the cluster size is fixed as I did not find a way how to put number of pets into environment variable. The proposal [1] states existence of ``IDENTITY_COUNT`` parameter as

```
- name: QUORUM_SIZE
  value: "$(IDENTITY_COUNT)"
```
Currently, this does not work.

[1] https://github.com/smarterclayton/kubernetes/blob/petset/docs/proposals/petset.md#parameterizing-pod-templates-and-supporting-resources